src/commands/protocols/publish/sender/publish-schedule-messages-command.js

```diff
@@ -76,6 +76,16 @@ class PublishScheduleMessagesCommand extends ProtocolScheduleMessagesCommand {
             agreementId,
         );
 
+        if (!agreementData) {
```

Comment:
Maybe this agreementData.scoreFunctionId === 0 should also this be added here?

Reply:
This agreement data ware read from blockchain, and if it has invalid data it will return null